core/tests/protractor_utils/AdminPage.js

```diff
@@ -152,6 +159,50 @@ var AdminPage = function() {
       'Could not set role successfully');
     return true;
   };
+
+  this.viewRolesbyRole = function(role) {
+    waitFor.visibilityOf(roleDropdown,
+      'View role dropdown taking too long to be visible');
+    roleDropdown.sendKeys('By Role');
+
+    roleValueOption.click();
+    roleValueOption.sendKeys(role);
+
+    viewRoleButton.click();
+    return true;
+  };
+
+  this.viewRolesbyUsername = function(username) {
+    waitFor.visibilityOf(roleDropdown,
+      'View role dropdown taking too long to be visible');
+    roleDropdown.sendKeys('By Username');
+
+    roleUsernameOption.click();
+    roleUsernameOption.sendKeys(username);
+
+    viewRoleButton.click();
+    return true;
+  };
+
+  this.expectUsers = function(expectedUsersArray) {
```

Comment:
This method name is incomplete, expect what?

Reply:
maybe change this to be expectUsersToMatch, or expectUsernamesToMatch

Reply:
Ok done!